modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -3,10 +3,15 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: CONCEPT
-[id="configuring_persistent_storage_{context}"]
+[id="configuring-persistent-storage_{context}"]
 = Configuring persistent storage
 
-Running cluster monitoring with persistent storage means that your metrics are stored to a persistent volume (PV) and can survive a pod being restarted or recreated. This is ideal if you require your metrics or alerting data to be guarded from data loss. For production environments, it is highly recommended to configure persistent storage. Because of the high IO demands, it is advantageous to use local storage.
+Run cluster monitoring with persistent storage to gain the following benefits:
+
+* Protect your metrics and alerting data from data loss by storing them to a persistent volume (PV). As a result, they can survive pods being restarted or recreated.
+* Avoid getting duplicate notifications and losing silences for alerts when the Alertmanager pods are restarted.
+
+For production environments, it is highly recommended to configure persistent storage. Because of the high IO demands, it is advantageous to use local storage.
```

Comment:
```suggestion
For production environments, it is highly recommended to configure persistent local storage.

Reply:
Not sure if the sentences above could be combined as they are both recommended?

Reply:
I did not want to touch the last sentence, because it is going to be deleted as part of another issue :D

Reply:
(We do not want to recommend using local storage anymore)